packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -115,14 +115,13 @@ export default class ClickableBehavior extends React.Component<Props, State> {
     }
 
     handleClick = (e: SyntheticMouseEvent<>) => {
-        if (this.props.onClick) {
+        if (this.keyboardClick && e) {
```

Comment:
Hmm - when would `e` be undefined? Shouldn't it always be defined? (especially as stated by the Flow rule)

Reply:
Double jk, it was actually because I wasn't passing in an object in the tests.